drive/modules/listener/types.bal

```diff
@@ -107,18 +107,18 @@ type StartPageTokenResponse record {
     string startPageToken;
 };
 
-# A change to a file or shared drive.
+# Represents the change of a file or shared drive.
```

Comment:
@SkNuwanTissera Can you use older description `A change to a file or shared drive.`

Reply:
Changed /Thanks